Gemfile

```diff
@@ -1,3 +1,7 @@
 source "https://rubygems.org"
 
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```

Comment:
Why do we need this?

Reply:
It is actually generated by Bundle. We get these gems from RubyGems, not from GitHub, so we don't need that.